build-tools-internal/version.properties

```diff
@@ -1,5 +1,5 @@
 elasticsearch     = 8.15.0
-lucene            = 9.10.0
+lucene            = 9.11.0-snapshot-d433394b292
```

Comment:
might wanna flip this on the Lucene RC build (rather than a snapshot)

Reply:
OK, I tried this, and added the RC location as mvn location, but now ES is looking in that location for ALL packages, and its failing the build. Any ideas on how to simply add a secondary mvn location to look for packages instead of overriding?